### PR TITLE
data: MuthoniMN

### PR DESCRIPTION
**Pull request by MuthoniMN**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: data/MuthoniMN.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    },
+    {
+      "icon": "FaTwitter",
+      "url" : "https://twitter.com/codebymitch"
+    },
+    {
+      "icon": "FaInstagram",
+      "url" : "https://www.instagram.com/codebymitch/"
+    }
+  ]
```
Comment (kumarsonsoff3) [inline data/MuthoniMN.json:29]: Please add the missing closing braces `}` in the end.
```suggestion
  ]
}
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

File: data/MuthoniMN.json (hunk 1 above)
Comment (MuthoniMN) [reply data/MuthoniMN.json:29]: I've added the closing brace. Please review my code again

**Review by sital002 — APPROVED** [review]
Looks good 👍
Welcome to Eddiehub community

**Comment by MuthoniMN** [comment]
> Looks good 👍 Welcome to Eddiehub community

Thank you🙏🏽

**Pull request merged**